openapi/src/main/openapi/sso-group-mapping/components.yaml

```diff
@@ -0,0 +1,116 @@
+responses:
+  SsoGroupMapping:
+    type: object
+    description: Description of an SSO group mapping
+    required:
+      - id
+      - group
+      - type
+    properties:
+      id:
+        type: string
+        description: SSO group mapping ID
+        example: "0164293d-d15a-465f-8f40-41a2133fb35e"
+      group:
+        type: string
+        description: SSO group name
+        example: "Marketing"
+      type:
+        type: string
+        description: Resource type
+        example: "sso-group-mapping"
```

Comment:
:thinking: 

type: enum
and possible value: `"sso-group-mapping"`?

Reply:
```
type: string
enum: ["sso-group-mapping"]
```